### DISCUSSION TITLE
how to check the display of snackbar( flutter ) with appium

### DISCUSSION THREAD

**Discussion by HamzaMiz** [appium/appium#18040] [Q&A]
Hello  , the snackbar itself is not visible in the Dom of the inspector , i searched on google and  found a solution but i think its for android native 
FluentWait<WebDriver> wait = new WebDriverWait(driver, 10, 100).ignoring(NoSuchElementException.class);
wait.until(visibilityOfElementLocated(By.id("snackbar_text")));
but my app under test is a  flutter app i tried that and it didnt work 
if anyone knows a solution enlighten me please and thanks

**Comment by Bigyama256** [comment]
Any work around for this ??
